src/index.js

```diff
@@ -0,0 +1,72 @@
+document.addEventListener("DOMContentLoaded", () =>  {
+
+  const url = "http://localhost:3000/pups"
+  const dogBar = document.querySelector("#dog-bar")
+  const dogInfo = document.querySelector("#dog-info")
+
+  fetch(url)
+  .then ((response) => response.json())
+  .then((dogData) => dogData.forEach((dog) => renderDogBar(dog)))
+
+  function renderDogBar(dog) {
+
+    const span = document.createElement("span")
+    const img = document.createElement("img")
+    const h2 = document.createElement("h2")
+    const goodButton = document.createElement("button")
+
+    span.textContent = `${dog.name}`
+    span.style.cursor = "pointer"
+
+    dogBar.appendChild(span)
+
+    span.addEventListener("click" , () => {
+      showDog(dog)
+    })
+
+    function showDog(dog) {
+
```

Comment:
You shouldn't have an empty line after function definition. Try to break this habit going forward.